### PR TITLE
Autofill the.env file with the "composer --no-interaction" command

### PR DESCRIPTION
**Pull request by roukmoute**
Currently, our production is runned without a .env.  
All is in the current environment variables.

So in production mode, we do not want to run want to answer to questions, just copy it.

This is an example of the result:
```bash
▶ composer install --no-interaction
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update

> Companienv\Composer\ScriptHandler::run
It looks like you are missing some configuration (46 variables). I will help you to sort this out.
Automatically confirmed in non-interactive mode

> MySQL

Automatically returned "mysql" in non-interactive mode
Automatically returned "my_database" in non-interactive mode
Automatically returned "root" in non-interactive mode

> Mailer

Automatically returned "mailhog" in non-interactive mode
Automatically returned "1025" in non-interactive mode
Automatically returned "" in non-interactive mode
Automatically returned "" in non-interactive mode

> RabbitMQ

Automatically returned "rabbitmq" in non-interactive mode
Automatically returned "5672" in non-interactive mode
Automatically returned "guest" in non-interactive mode
Automatically returned "guest" in non-interactive mode
Automatically returned "/" in non-interactive mode

> JWT

Automatically returned "foobar" in non-interactive mode

> Google API key

Automatically returned "MyGoogleAPIKey" in non-interactive mode
…
```

### CODE REVIEW

**Comment by sroze** [comment]
But if you do this... in development the plugin won't run either, right? 🤔

**Comment by roukmoute** [comment]
Of course !
I must check if the file exist.

But I don't like the name of this option, do you have any suggestion?

`ignore-dist-file-if-not-exist`?! :smile:

**Comment by sroze** [comment]
Let's rewind. Why don't you want the plugin to run in production? Because AFAIK, you will load the `.env` file only if your environment variables are not already defined, right? For example, Symfony will not load the `.env` file if the `APP_ENV` environment variable is already defined.

**Comment by roukmoute** [comment]
So you prefer I use an equivalent like Symfony?
I remove option and use only anything like `$_SERVER['APP_ENV']`.

For me your idea is clearly better, more simple and logic.

**Comment by roukmoute** [comment]
Up @sroze

**Comment by sroze** [comment]
What I meant was... why do you care if this script is run when your application won't use the `.env` file that was generated? 🤔

File: src/Companienv/Composer/InteractionViaComposer.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 use Companienv\IO\Interaction;
 use Composer\IO\IOInterface;
+use function sprintf;
```
Comment (sroze) [inline src/Companienv/Composer/InteractionViaComposer.php]: I don't think you need this 🤔

File: src/Companienv/Composer/InteractionViaComposer.php [hunk 2]
```diff
@@ -16,11 +17,23 @@ public function __construct(IOInterface $io)
...
+            return true;
+        }
+
         return $this->io->askConfirmation($question);
     }
 
     public function ask(string $question, string $default = null): string
     {
+        if (!$this->io->isInteractive()) {
+            $this->writeln(sprintf('Automatically returned %s in non-interactive mode', $default));
```
Comment (sroze) [inline src/Companienv/Composer/InteractionViaComposer.php]: Can you add quotes (`"`) around `%s`?

**Review by sroze — CHANGES_REQUESTED** [review]

File: src/Companienv/Composer/InteractionViaComposer.php (hunk 1 above)
Comment (roukmoute) [reply src/Companienv/Composer/InteractionViaComposer.php]: Adding this speed up resolving.

**Comment by sroze** [comment]
Thank you @roukmoute.

**Pull request merged**